strongbox-commons/src/main/resources/application.properties

```diff
@@ -22,3 +22,9 @@ java.io.tmpdir=strongbox-vault/tmp
 ehcache.disk.store.dir=strongbox/cache
 
 spring.servlet.multipart.enabled=false
+
+management.endpoints.web.exposure.include=health,info,beans,metrics,trace
+management.endpoint.health.show-details=always
+management.endpoint.shutdown.enabled=true
+management.endpoints.web.base-path=/api/monitoring
+management.security.role=ADMIN
```

Comment:
Cosmetic: Could you please make sure that resource files such as these end with a new line (some consoles and editors could have an issue with this).